applications/nrf5340_audio/doc/fota.rst

```diff
@@ -68,9 +68,17 @@ The first part of these names is based on :kconfig:option:`CONFIG_BT_DEVICE_NAME
    This is due to restrictions in the DFU system, and the error print is expected.
    The DFU process should still complete successfully.
 
+Building the FOTA configuration
+*******************************
+
+Use the :ref:`nrf53_audio_app_building_standard` procedure to build the nRF5340 Audio applications with the FOTA configuration.
+Make sure to provide the :ref:`correct configuration file <nrf53_audio_app_building_config_files>` :file:`prj_fota.conf` when running the build command.
+
+The :ref:`script-based method <nrf53_audio_app_building_script_running>` does not support building and programming the FOTA upgrades.
```

Comment:
Should we link to some pm_static.yaml description? Getting this set up right is important.

Reply:
We have this caution earlier in the configuration section

> .. caution::
   The application is provided with a memory partition configuration in :file:`pm_static_fota.yml`, which is required to perform FOTA using an external flash.
   The partition configuration is an example that can be changed between versions, and can be incompatible with existing devices.
   For this reason, always create a partition configuration that suits your application.
   See :ref:`partition_manager` for more information on how to create a partition configuration.